tests/subsys/bluetooth/mesh/light_ctrl/src/main.c

```diff
@@ -10,7 +10,6 @@
 #include <bluetooth/mesh/models.h>
 #include <bluetooth/mesh/light_ctrl_srv.h>
 #include <bluetooth/mesh/light_ctrl_reg_spec.h>
-#include "light_ctrl_internal.h"
```

Comment:
Was this just never used? Or is there a `prop_format_get` call I can't find that you changed as well?

Reply:
Never used! Or it was used at some time then the functionality was removed.

Reply:
Found that light_ctrl_internal was included to get subsys/.../sensor.h. Changed to directly including sensor.